src/ProgressOnderwijsUtils/Data/ParameterizedSqlObjectMapper.cs

```diff
@@ -399,22 +397,25 @@ static TRowReader<T> CreateLoadRowMethod<T>([NotNull] Func<ParameterExpression,
                 return ConvertLambdaExpressionIntoDelegate<T, TRowReader<T>>(loadRowsLambda);
             }
 
+            // Late-bound so we can generate faster code on .NET Framework while remaining compatible with .NET Standard.
+            static readonly MethodInfo compileToMethod = typeof(LambdaExpression).GetMethod(
+                "CompileToMethod", new[] { typeof(MethodBuilder) }
+                );
+
             [NotNull]
             static TDelegate ConvertLambdaExpressionIntoDelegate<T, TDelegate>([NotNull] Expression<TDelegate> loadRowsLambda)
             {
                 try {
-#if NET461
-                    if (typeof(T).IsPublic) {
+                    if (typeof(T).IsPublic && compileToMethod != null) {

```

Comment:
De oude check was sneller en minder code; ik denk niet dat dit beter is.

Reply:
Maakt het echt uit? Resultaat wordt gecachet toch?